src/components/Dialogs/Dialogs.js

```diff
@@ -12,8 +12,20 @@ const Dialogs = (props) => {
   const { dialogs, messages } = state;
 
 
-  const dialogsElements = dialogs.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
-  const messagesElements = messages.map((message) => <Message message={message.message} />);
+  const dialogsElements = dialogs.map((dialog) => (
```

Comment:
Consider using destructuring of "dialog" argument to get the "name" and "id" right in the "(...) =>" part.

Reply:
/fxd